Sources/BrowserServicesKit/ContentBlocking/AdClickAttribution/AdClickAttributionLogic.swift

```diff
@@ -173,6 +179,10 @@ public class AdClickAttributionLogic {
         guard case .activeAttribution(let vendor, let session, let rules) = state else {
             return
         }
+
+        if tld.eTLDplus1(host) == vendor {
+            counter.onAttributionActive()
+        }
```

Comment:
Not all navigation (e.g. going back) are calling this - are we concerned? I think it should not matter much if we are mostly interested in trends.

Reply:
I'd say not really - since we're interested in "loads" - perhaps going back just taking page straight from the cache. The problem with putting additional calls may result in duplicating events when both didFinishNav and onBackForwardNav are called. We would have to be smarter about it but it would result in more complicated code. I'm open though to the discussion.